src/applications/proxy-rewrite/redirects/crossDomainRedirects.json

```diff
@@ -1098,5 +1098,46 @@
     "domain": "www.benefits.va.gov",
     "src": "/gibill/vrrap.asp",
     "dest": "/education/other-va-education-benefits/veteran-rapid-retraining-assistance/"
+  },
+  {
+    "domain": "www.altoona.va.gov",
+    "src": "/",
+    "dest": "/altoona-health-care/",
+    "catchAll": true
```

Comment:
Is there any existing redirects that act as catch all or is this a first of it's kind?

Reply:
First of its kind unfortunately, this PR scares me quite a bit lol

Reply:
Yeah, this is one of those like, okay, _we've tested locally, now we gotta push to see it live_...

Reply:
This one scares me, too. Would it be better to merge after the daily deploy goes out so we have a good chunk of time to test before this goes to prod? Do you have a plan for testing this to make sure things are OK before this change hits prod?

Reply:
I won't be merging this until tomorrow. I'm not entirely sure how to test this fully before it goes to production tbh 🤔 I'm trying to think of ways currently

Reply:
Hey @k80bowman I added unit tests that fully test catch-all redirects, so feeling much more confident about this now! 🔥

Reply:
Fantastic! 🏆